ddtrace/contrib/redis/util.py

```diff
@@ -41,7 +45,10 @@ def format_command_args(args):
     out = []
     for arg in args:
         try:
-            cmd = stringify(arg)
+            if isinstance(arg, bytes):
+                cmd = arg.decode()
+            else:
+                cmd = stringify(arg)
```

Comment:
```suggestion
            cmd = ensure_text(arg)
```
?

Reply:
Using ensure_text here raises an exception on python 2. With this change we are seeing errors in most of our snap tests

Reply:
@mabdinur The [test run](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/12664/workflows/d6d82500-576f-4e29-b2a1-3198218bf755/jobs/917997/tests#failed-test-1) you linked is for redis rather than for aioredis. Can you point out the change that's causing the old tests to start failing?

I agree with @Kyle-Verhoog above that we should not introduce a change to the shared redis code in a feature PR like this.

Reply:
@majorgreys This change introduced the issue: https://github.com/DataDog/dd-trace-py/commit/005f9d33ea62365731909f6790fec65b6ac29029. Which is caused by using `ensure_text` instead of:
```
if isinstance(arg, bytes):
                cmd = arg.decode()
            else:
                cmd = stringify(arg)
``` . 
I reverted this suggestion but I am still curious why it occurs